backend/package.json

```diff
@@ -3,13 +3,24 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "node ./bin/www"
+    "start": "node ./bin/www",
+    "lint": "eslint --fix"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": [
+        "npm run lint -- --fix",
+        "git add"
```

Comment:
Hey, I don't think git add is necessary here since this will be calling only during commits. git add would have happened before the commit. Also, can you set up commit lint as well if possible? Let me know if you need resources for that :)

Reply:
Okay, I will make those changes and will let know.

Reply:
Do you mean commitlint as in this [https://github.com/conventional-changelog/commitlint](https://github.com/conventional-changelog/commitlint) ?

Reply:
Yes